client/my-sites/domains/domain-management/components/breadcrumbs/style.scss

```diff
@@ -190,3 +188,15 @@
 		display: flex;
 	}
 }
+
+.breadcrumbs__spacer {
+	height: 58px;
+
+	@include breakpoint-deprecated( '<960px' ) {
+		height: 67px;
+	}
```

Comment:
I was wondering about this intermediary breakpoint - do we need it? Or do we need only one height for mobile and one for desktop views?

Reply:
If we only need these two, we could use the `@include break-mobile` media query instead of the `breakpoint-deprecated` one, something like this:
```
.breadcrumbs__spacer {
  height: 76px;

  @include break-mobile {
    height: 58px;
  }
}
```

Reply:
Yes we need this because the #content wrapper change padding value at 960px, from 71px to 79px, and the spacer needs to compensate this difference (there are some weirds changes in the UI depending on old mq 😅). BTW I change from 67px to 66px.

Reply:
Got it, good call!